frontend/components/Dashboard/Nav/NavLinks.tsx

```diff
@@ -174,22 +173,13 @@ const NavLinks: React.FC<Props> = ({ onClick, currentUser }) => {
 
         :global(.expanded) .nav-bottom hr {
           width: 180px;
-          margin: 0 auto;
+          margin: 20px auto;
```

Comment:
The `<hr />` had no top/bottom padding and looked weird when the link above and below it were `active`. Adding margin also aligns it with the designs more